src/test/java/com/notably/logic/SuggestionTestUtil.java

```diff
@@ -0,0 +1,157 @@
+package com.notably.logic;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.notably.commons.path.AbsolutePath;
+import com.notably.commons.path.exceptions.InvalidPathException;
+import com.notably.model.Model;
+import com.notably.model.ModelManager;
+import com.notably.model.block.Block;
+import com.notably.model.block.BlockImpl;
+import com.notably.model.block.BlockModel;
+import com.notably.model.block.BlockModelImpl;
+import com.notably.model.block.Title;
+import com.notably.model.suggestion.SuggestionItem;
+import com.notably.model.suggestion.SuggestionItemImpl;
+import com.notably.model.suggestion.SuggestionModel;
+import com.notably.model.suggestion.SuggestionModelImpl;
+import com.notably.model.viewstate.ViewStateModel;
+import com.notably.model.viewstate.ViewStateModelImpl;
+
+/**
+ * Contains helper methods for testing suggestion feature.
+ */
+public class SuggestionTestUtil {
+    private static AbsolutePath toCs2103;
+    private static AbsolutePath toCs3230;
+    private static AbsolutePath toCs2103Week1;
+    private static AbsolutePath toCs2103Week2;
+    private static AbsolutePath toCs2103Week3;
+    private static AbsolutePath toCs2103Week1Lecture;
+    private static Model model;
```

Comment:
Hmm tbh, I don't really like these static variables over here. Using static fields means that you'd only have one instance of `toCs2103` (for example) per JVM, which I don't think is right since we would like to isolate each test class as much as possible (even though JUnit actually runs test cases sequentially, so this would still perform as expected).

Reply:
ok will update!